### ISSUE TITLE
LOD offset error when opening Shogun 2 rigidmodelv2 files

### ISSUE THREAD

**Issue by ComradeCheekiBreeki** [Frodo45127/rpfm#252]
**Describe your problem**
Attempting to access/open Shogun 2 `rigid_model_v2` files throws an error saying the LOD header offset is incorrect (I've encountered a similar error on other programs designed for viewing/exporting rigid models). I suspect the file format is different between the games, as all the tools in the last few years are geared towards Warhammer or Rome II+. I haven't bothered to look at the file in a hex editor yet but even if I did I don't think I'd know how to fix the code.

The inability to view or export rigid model files from Shogun 2 into a format readable by Blender has been a thorn in my side for ages, since I can't and won't spend the money required to purchase 3DSMax, which is the only program that has complete support for rigid model files as the converter is included in the Assembly Kit. Otherwise I've found no other program, script, plugin etc on the internet that allows me to do this so for the time being I'm just unable to deal with models in any way.

**Extra info?**
RPFM version: 4.0.19 (on beta branch)

**Comment by Frodo45127** [comment]
yeah, that's.... hard to fix. The rigidmodel module is a separate lib for which the source code got lost, so I cannot update it. And not sure when will be re-done.
